examples/tensorflow/classification/main.py

```diff
@@ -147,6 +147,8 @@ def load_compression_state(ckpt_path: str):
     return checkpoint.compression_state.state
 
 
+# pylint: disable=R0913
+# pylint: disable=R0915
```

Comment:
The pylint error R0913- too-many arguments should have appeared before and it happens because `train_epoch_fn `has many arguments. To avoid this it needs to refactor the whole main.py. 
The same for R0915 - too-many statements. 
As I know we plan to refactor samples and divide them into pieces, so I think that now it doesn't make sense to try to refactor them. However, if you think that it is necessary now I will do it

I am waiting for your opinion

Reply:
Done